### PR TITLE
Add storageclass documentation for APPUiO Cloud zones on Exoscale

### PR DESCRIPTION
**Pull request by 54nd20**
## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/explanation/storage-classes.adoc [hunk 1]
```diff
@@ -59,3 +59,28 @@ image::storage_performance/cloudscale_read_bandwidth.png[]
 
 image::storage_performance/cloudscale_write_iops.png[]
 image::storage_performance/cloudscale_write_bandwidth.png[]
+
+== exoscale.com
```
Comment (tobru) [inline docs/modules/ROOT/pages/explanation/storage-classes.adoc:63]:
```suggestion
== Exoscale
```

Please take care of spelling the Company name correctly

File: docs/modules/ROOT/pages/explanation/storage-classes.adoc [hunk 2]
```diff
@@ -59,3 +59,28 @@ image::storage_performance/cloudscale_read_bandwidth.png[]
...
+
+On exoscale.com we provide RWO storage using the https://products.docs.vshn.ch/products/appuio/managed/storage_cluster.html[APPUiO Managed Storage Cluster].
```
Comment (tobru) [inline docs/modules/ROOT/pages/explanation/storage-classes.adoc:65]:
```suggestion
On Exoscale we provide RWO storage using the https://products.docs.vshn.ch/products/appuio/managed/storage_cluster.html[APPUiO Managed Storage Cluster].
```

File: docs/modules/ROOT/pages/explanation/storage-classes.adoc [hunk 3]
```diff
@@ -59,3 +59,28 @@ image::storage_performance/cloudscale_read_bandwidth.png[]
...
+Additionally, we provide RWX storage on CephFS managed via https://products.docs.vshn.ch/products/appuio/managed/storage_cluster.html[APPUiO Managed Storage Cluster].
+
+The following storage classes are available on exoscale.com:
```
Comment (tobru) [inline docs/modules/ROOT/pages/explanation/storage-classes.adoc:68]:
```suggestion
The following storage classes are available:
```

**Review by tobru — CHANGES_REQUESTED** [review]

**Review by simu — APPROVED** [review]
Directly adjusted the text a bit and added the performance section for Exoscale using data from the ch-gva-2 zone (RBD read IOPS are a significantly different from the ones in the product docs, not sure yet why).

**Pull request merged**